### DISCUSSION TITLE
Linked contacts and quick dial

### DISCUSSION THREAD

**Discussion by ghost** [FossifyOrg/General-Discussion#676] [Q&A] [answered]
Hi. Does any body know the following?

1) Is there a feature like other contacts app? where we press and hold the Contacts icon to see favorite contacts appear for fast dialing without opening the app.

2) Is there a feature to link contacts like? Linking related contacts to easily find them from a single contact.

**Comment by Aga-C** [comment, marked as answer]
1. No.
2. There is a feature request for it: https://github.com/FossifyOrg/Contacts/issues/90.

**Reply by ghost** [reply]
> * No.

How do I open a feature request for this? I want to request for this across Fossify Contacts, Messages and Phone. Can you open a common request somehow?

**Reply by Aga-C** [reply]
You can create a feature request here: https://github.com/FossifyOrg/Contacts/issues/new?assignees=&labels=feature+request%2Cneeds+triage&projects=&template=feature_request.yml. Phone and Messages already have feature requests for it.

**Reply by ghost** [reply]
> You can create a feature request here: https://github.com/FossifyOrg/Contacts/issues/new?assignees=&labels=feature+request%2Cneeds+triage&projects=&template=feature_request.yml. Phone and Messages already have feature requests for it.

If phone already have then no need for me to do for contacts as we dial mostly from phone. Thanks for your help.
